src/core/core.scale.js

```diff
@@ -654,14 +667,24 @@ module.exports = Element.extend({
 			if (maxTicks && tickCount > maxTicks) {
 				skipRatio = Math.max(skipRatio, Math.floor(tickCount / maxTicks));
 			}
+
+			// Requesting a label spacing cancels other settings established by maxTicks
+			if (labelSpacing) {
+				skipRatio = labelSpacing;
+			}
 		}
 
 		for (i = 0; i < tickCount; i++) {
 			tick = ticks[i];
 
-			// Since we always show the last tick,we need may need to hide the last shown one before
-			shouldSkip = (skipRatio > 1 && i % skipRatio > 0) || (i % skipRatio === 0 && i + skipRatio >= tickCount);
-			if (shouldSkip && i !== tickCount - 1) {
+			if (evenLabelSpacing || labelSpacing) {
+				if (skipRatio > 1 && i % skipRatio > 0) {
+					// leave tick in place but make sure it's not displayed (#4635)
+					delete tick.label;
+				}
+			} else if ((i !== tickCount - 1) && ( // Always show last tick
+				(skipRatio > 1 && i % skipRatio > 0) ||
```

Comment:
I would indent this line and the next one more tab so that it doesn't line up with the content inside the curly braces (assuming that doesn't cause the linter to fail)